boards/posix/native_posix/doc/board.rst

```diff
@@ -373,6 +378,10 @@ The following peripherals are currently provided with this board:
 
 **Entropy device**:
   An entropy device based on the host :c:func:`random` API.
+  This device will generate the same sequence of random number if initialized
+  with the same random seed.
+  To change this random seed use the command line option
```

Comment:
Is the random seed a decimal number?  What range of values?  Can it be negative?  (Is this explained if you use ``zephyr/zephyr.exe --help``)?

Reply:
Thanks David. It is a 32 bit unsigned integer, so any integer number in the range [0..2^32-1] is acceptable. A negative number is actually also taken without complaint (it will be assigned modulo 2^32 to the variable). The number can be given in decimal, hex or octal.
Would you like this described in the doc? (if so, feel free to propose the changes)

As of now `--help` just prints:
```
$ zephyr/zephyr.exe --help
[-h] [--h] [--help] [-?] [-stop_at=<time>] [-seed=<r_seed>] [-testargs <arg>...] 

 [-h] [--h] [--help] [-?]     :Display this help
 [-stop_at=<time>]            :In simulated seconds, when to stop automatically
 [-seed=<r_seed>]             :Seed for the entropy device
 [-testargs <arg>...]         :Any argument that follows will be ignored by the top level, and made 
                              available for possible tests
```
Please tell if you would like some correction or addition in this help message.

Reply:
We do need to document the <r_seed> value somewhere, and in particular the command line format and value range.  So I'd add something to the ``--help`` output for the seed option, maybe:

    A 32-bit integer seed value for the entropy device, such as 97229 (decimal),
    0x17BCD (hex), or 0275715 (octal)

And also change the text in this documentation to read:

    You can change this random seed value by using the command line option:
    ``--seed=<random_seed>`` where the value specified is a 32-bit integer
    such as 97229 (decimal),  0x17BCD (hex), or 0275715 (octal).